lib/features/recurring_donations/create/widgets/create_recurring_donation_bottom_sheet.dart

```diff
@@ -68,6 +72,12 @@ class _CreateRecurringDonationBottomSheetViewState
     final currencySymbol = NumberFormat.simpleCurrency(
       name: country.currency,
     ).currencySymbol;
+
+    if (user.country != Country.us.countryCode ||
```

Comment:
just a thoughts: we have so many places with such or very similar code. idk how yet, but it would be great to incapsulate such logic at leas in Util class, but even there I think we could come up with some more generic solution

Reply:
Yes! I agree, let's make it as an improvement on the backlog